jd-geek-source/src/app/components/icons/angle-right-icon/angle-right-icon.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from "@angular/core";
+import { faAngleRight } from "@fortawesome/free-solid-svg-icons";
+
+@Component({
+  selector: "app-angle-right-icon",
+  templateUrl: "./angle-right-icon.component.html",
+  styleUrls: ["./angle-right-icon.component.scss"],
+})
+export class AngleRightIconComponent implements OnInit {
+  faAngleRight = faAngleRight;
+
+  constructor() {}
+
+  ngOnInit(): void {}
```

Comment:
if you're not going to use onInit, you can just get rid of it, the constructor, and the "implements OnInit" above for a cleaner look.